samples/tableadmin.js

```diff
@@ -0,0 +1,289 @@
+/**
+ * Copyright 2018, Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+// Imports the Google Cloud client library
+const Bigtable = require('@google-cloud/bigtable');
+
+async function runTableOperations(instanceName, tableName) {
+    // const instanceName = "my-instance";
+    // const tableName = "my-bigtable-name";
+    const bigtableClient = new Bigtable();
+    const instance = bigtableClient.instance(instanceName);
+    const table = instance.table(tableName);
+
+    // Check if table exists
+    console.log('\nChecking if table exists...');
+    let tableExists;
+    try {
+        [tableExists] = await table.exists();
+    } catch (err) {
+        console.error(`Error retrieving status of table ${err}`);
+        return;
+    }
+
+    if (!tableExists) {
+        try {
+            // Create table if does not exist
+            console.log(`Table does not exist. Creating table ${tableName}`);
+            // Creating table
+            await table.create();
+        } catch (err) {
+            console.error(`Error creating table ${err}`);
+            return;
+        }
+    } else {
+        console.log(`Table exists.`);
+    }
+
+    console.log('\nListing tables in current project...');
```

Comment:
`\n` isn't needed here. You can add an empty call to `console.log()` to make a newline on the console